Dockerfile

```diff
@@ -1,10 +1,10 @@
-FROM crystallang/crystal:1.10.1-alpine as builder
+FROM crystallang/crystal:1.12-alpine as builder
```

Comment:
```suggestion
FROM crystallang/crystal:1.14-alpine as builder
```

This should fix the error in https://app.circleci.com/pipelines/github/codacy/codacy-ameba/142/workflows/fd56e448-f53d-4144-95a1-2365b631c925/jobs/511